ui/pages/add-token/add-token.component.js

```diff
@@ -236,10 +247,16 @@ class AddToken extends Component {
       customAddressError,
       customSymbolError,
       customDecimalsError,
-      autoFilled,
       forceEditSymbol,
+      symbolAutoFilled,
+      decimalAutoFilled,
     } = this.state;
 
+    const { chainId } = this.props;
+    const blockExplorerTokenLink = createTokenTrackerLinkForChain(
```

Comment:
I have an open PR to bump our etherscan-link version and make use of a new method `getTokenTrackerLink`, in which you can pass the RPCPrefs to get custom network block explorer link generation. This will probably land first though so we can change this to use the new method later on I suppose.